services/ui-src/src/additional-scss/_header.scss

```diff
@@ -40,3 +42,7 @@
     @extend .font-alt-lg;
   }
 }
+
+.usa-banner__inner {
+  margin: 0;
+}
```

Comment:
I don't know about this one. I checked cms.gov, hhs.gov, irs.gov, va.gov, ed.gov... They all have the same banner, with the same wonky margin on the left side.

Reply:
i will revert it begrudgingly 😞